.githooks/pre-commit

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+#!————————————————————————————————————————————————————————————————————————————*!
+# Git pre-commit hook.
+#
+# Run `make setup-hooks` to install.
+#!————————————————————————————————————————————————————————————————————————————*!
+
+echo "running pre-commit hook..."
+
+# Format files.
+make clean
```

Comment:
Need to re-add the files that have been formatted so that they're included in the commit - a simple git add . should do

Reply:
good catch